shared/src/price_estimation/gas.rs

```diff
@@ -0,0 +1,21 @@
+///! constants to estimate gas use in GPv2
+
```

Comment:
ubernit:

can we just define a constant:
```
pub SINGLE_TRADE_OVERHEAD = SETTLEMENT + TRADE + 2*ERC20_TRANSFER
```

instead of doing the arithmetic everywhere its used? That way, for all our current estimators we can use `gas::SINGLE_TRADE_OVERHEAD` and, in the future if it we have an estimator that behaves differently, we can use the other exported constants?

Reply:
done